app/javascript/components/components/Header.js

```diff
@@ -16,7 +17,8 @@ class Header extends Component {
       <>
         <h1>nest.</h1>
         <Nav>
-          <NavItem><Link src={Home} className="nav-link">Home</Link></NavItem>
+          <NavItem><Link to="/" className="nav-link">Home</Link></NavItem>
+          <NavItem><Link to="/apartmentindex" className="nav-link">Apartments</Link></NavItem>
```

Comment:
I would suggest a small cleanup here of dropping your nested `Link` inside the `NavItem` to the next line to match the format of the rest of this component.